### DISCUSSION TITLE
DOI not linking correctly to webpage ?

### DISCUSSION THREAD

**Discussion by ChrisDony** [knowledgefutures/pubpub#2203] [Support]
Hi,

I've stated creating a community and created two pubs with doi:
https://christophedony.pubpub.org/pub/scholcomm1/release/3
https://christophedony.pubpub.org/pub/pub8bfnebv0/release/1

Both of the assigned DOIs have been registered at crossref (https://doi.org/10.21428/352619e2.ff083dcc and https://doi.org/10.21428/352619e2.3958d561) but when if I click on those or copy/paste them, I get a "Not Found" page result.

I'm not sure exactly if I did anything wrong for this to happen (getting started with the pub pub platform). I have checked discussion 
https://github.com/pubpub/pubpub/discussions/1272 and realize I may have changed the url name (but I'm not sure anymore).

Is there any way I can fix this so as to not make the same mistake again in the future? 

Thanks for your help and time already !

Chris

**Comment by gabestein** [comment]
Hi Chris - first, I really liked your comic!

Second, yup, it’s the same thing as in #1272. The url changed after submission to crossref, so crossref had to be notified of the change. I was able to fix it by redepositing the DOI via Pub settings. If you run into DOI issues in the future, definitely try re-submitting, as that often resolves issues. Also, we can do a better job alerting users to changes that would break DOIs so they know to resubmit (or even automate resubmission). I’ll chat about it with the team.

**Reply by ChrisDony** [reply]
Hi,

Thanks for the quick fix ! 

And glad you enjoyed the comic !

Best
